### PR TITLE
[Cloud Security] add cloud_configuration_finding data stream to wiz

### PR DESCRIPTION
**Pull request by maxcold**
## Checklist

- [ ] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [ ] I have verified that all data streams collect metrics or logs.
- [ ] I have added an entry to my package's `changelog.yml` file.
- [ ] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Author's Checklist

- [ ] 

## Related issues

-

### CODE REVIEW

File: packages/wiz/data_stream/cloud_configuration_finding/_dev/test/pipeline/test-issue.log-expected.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
+{
```
Comment (maxcold) [inline packages/wiz/data_stream/cloud_configuration_finding/_dev/test/pipeline/test-issue.log-expected.json:1]: @maxcold rename the file to match the data stream name

File: packages/wiz/elasticsearch/transform/latest_csp_findings/transform.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
+source:
+  index:
+    - "logs-wiz.cloud_configuration_finding-*"
+dest:
+  index: "logs-wiz.cloud_configuration_finding_latest-default"
```
Comment (kfirpeled) [inline packages/wiz/elasticsearch/transform/latest_csp_findings/transform.yml:5]: omit `-default` from the destination index, it is preserved for namespaces. which is not relevant here

File: packages/wiz/elasticsearch/transform/latest_csp_findings/transform.yml [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+latest:
+  unique_key:
+    - event.id
```
[the remark this answers is not in the document]
Comment (maxcold) [reply packages/wiz/elasticsearch/transform/latest_csp_findings/transform.yml:8]: @kfirpeled yes, it should be a combination of the fields we care about same or similar to our native integration. I didn't intend this PR to be production ready and for POC it didn't make a difference really. there are a lot of things to fix if we want to release the `cloud_configuration_finding` data stream ourselves

File: packages/wiz/elasticsearch/transform/latest_csp_findings/transform.yml (hunk 2 above)
Comment (maxcold) [reply packages/wiz/elasticsearch/transform/latest_csp_findings/transform.yml:5]: @kfirpeled It wasn't really important for the POC, for better or for worse it shows that the namespace question is still open. I will investigate if we can get around the problem of `constant_keyword` and have namespace indifferent latest index or not

Comment (maxcold) [reply packages/wiz/elasticsearch/transform/latest_csp_findings/transform.yml:5]: as you suggested it should be possible to have `keyword` as a type for `data_stream.namespace` even though the schema restrict it with `constant_keyword`. Here is some discussion around why restrict it in the schema in the first place https://github.com/elastic/ecs/pull/845#issuecomment-631719002 . Though going against ECS might impose some risks, eg. if the package-spec implements some validation for that in the future for some reason

**Comment by maxcold** [comment]
Keeping this open as the POC it is a part of is still relevant. Will close after Wiz data stream is implemented

**Comment by maxcold** [comment]
closing as transform has already been released and this POC is not relevant enymore

**Pull request closed without merging**